mmf/modules/attention.py

```diff
@@ -156,3 +160,107 @@ def forward(self, image_feat, question_embedding, image_locs=None):
                 masked_attention = self._mask_attentions(attention, image_locs)
 
         return masked_attention
+
+
+class MultiHeadAttention(nn.Module):
```

Comment:
Use torch's multiheadattention implementation directly?

Reply:
We will replace this in next PRs as this changes our model weigths from the challenge. Same for other methods from torch transformer.